### DISCUSSION TITLE
GitHub Sponsors

### DISCUSSION THREAD

**Discussion by cleptric** [slack-go/slack#1335] [General]
Hey,

would you consider setting up GitHub Sponsors? We use this package at Sentry for one of our Slack Bots https://github.com/getsentry/gib-potato and I would like to route some of our yearly OSS funding towards this project.

**Comment by nlopes** [comment]
Hi @cleptric,

That's very kind of you and of Sentry. I'm setting this up in https://github.com/slack-go/slack/pull/1388.
Please have a look at the description there to make sure you/Sentry are aligned on providing sponsorship.

Happy for more nuanced views but for now I'm merging what I put there 🙏

**Reply by cleptric** [reply]
Sounds good, thank you. Next round of OSS funding will happen in the fall, will make sure to include you 🙂
